### PR TITLE
policy documentation

### PR DESCRIPTION
**Pull request by mbozwood**

### CODE REVIEW

File: policy/approved-software.mdx [hunk 1]
```diff
@@ -2,6 +2,8 @@
 title: Approved Software
 ---
 
+Current as of 01NOV2019
```
Comment (doylemark) [inline policy/approved-software.mdx:5]: Is this annotation still necessary? I feel this makes the site look somewhat "parked & abandoned". I think the reader should be able to assume all content on our website is up to date. The update date is recorded in our commit history anyway

File: policy/global-ratings-policy.mdx [hunk 2]
```diff
@@ -79,11 +79,11 @@ training, can be given temporary permission to practice solo on an Approach/Depa
 with the rating of Student 2 (S2). Such validations shall:
 
 1. Be limited to one training airport, and
-1. Be clearly listed on the local facility's website, detailing the member's name, VATSIM ID and the position(s) that they are validated on, and
-1. Have a maximum period of validity of 30 days, and
-1. Be subject to regular review, and
-1. Not be renewed/extended unless the candidate has made good use of the facility and has shown improvement over the period of validity of the solo endorsement, and
-1. Renewable for a total period of up to 90 days, with each renewal only to have a maximum period of validity of 30 days
```
[the remark this answers is not in the document]
Comment (doylemark) [reply policy/global-ratings-policy.mdx:86]: Not that it matters much

File: policy/global-ratings-policy.mdx [hunk 3]
```diff
@@ -195,317 +195,4 @@ can be removed at the discretion of the Region Director.
 * **6.7**   Each division should at all times prominently display on its web pages a list of authorised designated ”Major Airports” 
 and authorised designated “Special Centers” that exist within the division airspace.
 
-* **6.8**   A list of all current EC approved Designated Airspace is attached as Appendix B to this policy.
```
Comment (doylemark) [inline policy/global-ratings-policy.mdx:198]: Where is section 7 and below going?

File: policy/overview.mdx [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
```
Comment (doylemark) [inline policy/overview.mdx:1]: Good content. I feel perhaps this page and the list of BOG meetings could due with a custom component of some kind, these long lists sometimes don't look the best. v1.1 perhaps

**Review by doylemark — CHANGES_REQUESTED** [review]
Great stuff, thanks this is very much appreciated, especially the CoC is 👌 .

File: policy/approved-software.mdx (hunk 1 above)
Comment (mbozwood) [reply policy/approved-software.mdx:5]: It'll be updated in the coming weeks but I would say yes. I might event add a recent changes part to the page. 
It's not a date for us, you've rightly noted that we can see it it commit history but it's for others to see when things were last changed. I'll have a rethink on the page.

Comment (doylemark) [reply policy/approved-software.mdx:5]: cool

File: policy/global-ratings-policy.mdx (hunk 3 above)
Comment (mbozwood) [reply policy/global-ratings-policy.mdx:198]: fixed in #2a59ca0

**Review by doylemark — COMMENTED** [review]
review complete

**Review by doylemark — COMMENTED** [review]
review complete

**Review by doylemark — APPROVED** [review]

**Pull request merged**